runtime/lib/app-runtime.js

```diff
@@ -602,11 +602,14 @@ AppRuntime.prototype.setMicMute = function setMicMute (mute, options) {
   }
 
   return future
-    .then(() => this.component.light.play(
-      '@yoda',
-      'system://setMuted.js',
-      { muted: muted },
-      { shouldResume: muted }))
+    .then(() => {
+      var noTts = !!this.component.lifetime.activeSlots.scene
```

Comment:
> What's the reason of preferring checks on scene app rather than checks on active app by `this.component.lifetime.getCurrentAppId()`?

Yeah, the real intent is `No need to play tts when some apps are active`.